lighthouse-core/config/index.js

```diff
@@ -171,16 +172,79 @@ function filterAudits(audits, auditWhitelist) {
   return filteredAudits;
 }
 
-function expandAudits(audits) {
+function expandAudits(audits, paths) {
+  const rootPath = path.join(__dirname, '../../');
```

Comment:
This would be more clear to me if there is one loop which resolves our core audits, and then a second one afterwards which hunts down ones not resolved in the provided paths.